app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_height="match_parent"
+    android:layout_width="match_parent">
+
+
+<androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#191926"
+    tools:context=".MainActivity">
+
+    <ImageView
+        android:id="@+id/imageView4"
```

Comment:
@atashandroid Please add more understandable ids to the views (for example `backgroundImage`) In the future you will use these ids in code and it will be very hard to understand with naming like that (`imageView1`, `imageView2`)